lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -154,8 +159,14 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     ///
     /// Throws a connection_not_init_error in case the connection has
     /// not been opened previously.
+    /// Also rethrows any exception that can be thrown by
+    /// startMonitoring.
```

Comment:
Since we have a blocking call, we should rename start/stopMonitoring to something that gives a better idea of a non-blocking call. I'll deal with that later.